xyz.armcord.ArmCord.yml

```diff
@@ -0,0 +1,59 @@
+app-id: xyz.armcord.ArmCord
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+command: startarmcord
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland # End users can actually *use* the Wayland backend by setting the USE_WAYLAND environment variable to 1
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications # Discord push notifs
+  - --talk-name=org.kde.StatusNotifierItem-3-1 # Needed for the tray icon, at least supposedly
+  - --device=all # Needed for GPU acceleration and the webcam
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download # This and the above two are used for drag-n-drop, and download managing
+
+modules:
+  - name: minerals
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 minerals/startarmcord /app/bin/startarmcord
+    - install -D minerals/xyz.armcord.ArmCord.desktop -t /app/share/applications/
+    - install -D minerals/xyz.armcord.ArmCord.metainfo.xml -t /app/share/metainfo/
+    - install -D minerals/xyz.armcord.ArmCord-48.png /app/share/icons/hicolor/48x48/apps/xyz.armcord.ArmCord.png
+    - install -D minerals/xyz.armcord.ArmCord-64.png /app/share/icons/hicolor/64x64/apps/xyz.armcord.ArmCord.png
+    - install -D minerals/xyz.armcord.ArmCord-128.png /app/share/icons/hicolor/128x128/apps/xyz.armcord.ArmCord.png
+    - install -D minerals/xyz.armcord.ArmCord-256.png /app/share/icons/hicolor/256x256/apps/xyz.armcord.ArmCord.png
```

Comment:
Generally speaking, shipping only the biggest icon will do the trick.

Reply:
Is there anything to look out for when shipping only the largest one? I'm not sure if it's standard for application menus to downscale icons if they're too large.

Reply:
GNOME and KDE handles that just fine, and our tooling resizes the icon to 128x128 and 64x64 for internal usage.

Reply:
That's a nice surprise, I'll get on that soon-ish.